modules/nw-osp-loadbalancer-limitations.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+// * networking/load-balancing-openstack.adoc
+// For thinking and reviewing, adding to networking/load-balancing-openstack.adoc
+
+:_content-type: CONCEPT
+[id="nw-osp-loadbalancer-limitations_{context}"]
+= Limitations of load balancer Services
+
+{product-title} clusters on {rh-openstack-first} use Octavia to handle load balancer Services. You should be aware of some functional limitations that are a result of this.
+
+{rh-openstack} Octavia has two available providers: Amphora and OVN. These two differ in terms of available features as well as implementation details and these distinctions affect load balancer Services created in {product-title}.
+
+== `externalTrafficPolicy` set to `Local`
```

Comment:
It's a pain, but these two H2s will probably have to be broken into their own discrete modules to adhere to our mod docs style.

They could also become an unordered list, though the first section might be a bit long for that.

Do you have a preference for either approach as I review this? @dulek 

Pausing here.

Reply:
I don't have a preference, let me break it down.